workspace/src/main/resources/public/ts/delegates/actions/create.ts

```diff
@@ -118,6 +119,9 @@ export function ActionCreateDelegate($scope: CreateDelegateScope) {
             return isSharedTree;
         }
     }
+    $scope.isSharedTree = function () {
+        return $scope.currentTree.filter == "shared";
```

Comment:
De manière générale, on préfère l'opérateur `===` (en js et ts) pour vérifier l'égalité.
 - `==` vérifie uniquement que les valeurs sont égales (en castant les variables).
 - `===` est plus strict et vérifie que le type et la valeur sont égaux.

`==` introduit souvent des bugs car il existe des pièges au cast, ex. un certain nombre de valeurs en js qui valent `false` en booléen (https://developer.mozilla.org/fr/docs/Glossaire/Falsy https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Op%C3%A9rateurs_de_comparaison#Les_op%C3%A9rateurs_d'%C3%A9galit%C3%A9)

```suggestion
        return $scope.currentTree.filter === "shared";
```